### PR TITLE
seo fixes

### PR DESCRIPTION
**Pull request by MateuszUrbanowicz**

### CODE REVIEW

File: public/views/content/details/tabs/detailsEdit.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
                     </div>
 
                     <div class="col-md-5 text-right">
-                        <button ng-click="showSeoDetails = !showSeoDetails"
+                        <button ng-click="$root.showSeoDetails = !$root.showSeoDetails"
                                 type="button"
                                 class="btn btn-default">
                             {{ 'SHOW_SEO_SETTINGS' | translate }}
```
Comment (PSkierniewski) [inline public/views/content/details/tabs/detailsEdit.html]: When settings are showed, button text should be changed to "Hide SEO settings"

File: public/views/content/details/tabs/detailsEdit.html [hunk 2]
```diff
@@ -42,40 +42,45 @@
...
-                               type="text"/>
+                        <div class="col-md-12">
+                            <input ng-model="activeTranslation.seoDescription" class="form-control" id="contentSeoDescription"
+                                   type="text"/>
 
-                        <p><i>
-                            <small>{{ 'SEO_DESCRIPTION_DESCRIPTION' | translate }}</small>
-                        </i></p>
+                            <p><i>
+                                <small>{{ 'SEO_DESCRIPTION_DESCRIPTION' | translate }}</small>
```
Comment (PSkierniewski) [inline public/views/content/details/tabs/detailsEdit.html]: We should use Twitter Bootstrap "Help text" and using the aria-describedby attribute:

http://getbootstrap.com/css/#forms

File: public/views/content/details/tabs/detailsEdit.html [hunk 3]
```diff
@@ -42,40 +42,45 @@
...
-                        <input ng-model="activeTranslation.seoTitle" class="form-control" id="contentSeoTitle" type="text"/>
+                        <div class="col-md-12">
+                            <input ng-model="activeTranslation.seoTitle" class="form-control" id="contentSeoTitle" type="text"/>
 
-                        <p><i>
-                            <small>{{ 'SEO_TITLE_DESCRIPTION' | translate }}</small>
-                        </i></p>
-                    </div>
+                            <p><i>
+                                <small>{{ 'SEO_TITLE_DESCRIPTION' | translate }}</small>
```
Comment (PSkierniewski) [inline public/views/content/details/tabs/detailsEdit.html]: We should use Twitter Bootstrap "Help text" and using the aria-describedby attribute:

http://getbootstrap.com/css/#forms

**Comment by MateuszUrbanowicz** [comment]
@PSkierniewski ptal

**Comment by PSkierniewski** [comment]
:+1:

**Pull request merged**